crates/astria-sequencer-utils/src/config.rs

```diff
@@ -0,0 +1,17 @@
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+pub struct Config {
```

Comment:
If you add a doc comment above this type describing the app, then clap will automatically derive a nice help page from that.